extensions/ql-vscode/src/vscode-tests/no-workspace/remote-queries/variant-analysis-results-manager.test.ts

```diff
@@ -10,7 +10,7 @@ import * as path from 'path';
 import { VariantAnalysisResultsManager } from '../../../remote-queries/variant-analysis-results-manager';
 import { createMockVariantAnalysisRepoTask } from '../../factories/remote-queries/gh-api/variant-analysis-repo-task';
 import { CodeQLCliServer } from '../../../cli';
-import { storagePath } from '../global.helper';
+import { storagePath } from '../../cli-integration/global.helper';
```

Comment:
Should we still be relying on this file from the `cli-integration` directory, or should that file be moved to be somewhere more generic? Although I'm also aware we don't want to bloat the PR unless necessary.

Reply:
There are still three files in the cli-integration folder that depend on that helper: 

- `extensions/ql-vscode/src/vscode-tests/cli-integration/databases.test.ts`
- `extensions/ql-vscode/src/vscode-tests/cli-integration/new-query.test.ts`
- `extensions/ql-vscode/src/vscode-tests/cli-integration/queries.test.ts`

Reply:
I don't think the `storagePath` is set outside of the `cli-integration` tests (it's set in a helper that is only loaded by the `cli-integration` tests), so this would be `undefined` in these tests. Where are the results actually stored when running these tests now?

Reply:
It was actually failing even earlier, when it was trying to mock the extension. I've had to change the way we stub things (including the storage path): https://github.com/github/vscode-codeql/commit/ca3bf049a48aca4318d0c83583f8ce8912d47c27 and https://github.com/github/vscode-codeql/pull/1628/commits/37bd51cd9dbdff602af90cdc5632137dc91f3301.